tensorboard/BUILD

```diff
@@ -241,11 +241,23 @@ py_library(
     visibility = ["//visibility:public"],
 )
 
+config_setting(
+    name = "dev_build",
+    # consider changing this to fastbuild
+    values = {"compilation_mode": "dbg"},
+)
+
+config_setting(
+    name = "dev_flag",
```

Comment:
Do we need both config settings?  Would it be reasonable to just support the dbg option and not the custom flag as well?

Reply:
Update PR description?